### PR TITLE
Reorganise ops, processors and utils as directories

### PR DESCRIPTION
**Pull request by jakobsj**
This separates operators and processors into separate py files held in directories instead of a single file, for consistency with framework structure. Included __init__s to allow importing, as well as added packages in setup.py.

Updated all CCPi-astra demo to import from ccpi.astra.operators instead of old ccpi.astra.ops.

Still need to update CCPi-astra demos to use the new algorithms etc, but I'd suggest we merge this pull request initially, to allow the updated structure to be used in demos in the framework.

### CODE REVIEW

**Comment by jakobsj** [comment]
Wait, this is to address #17, but so far only the reorganisation is done, so still need to add/remove methods.

File: Wrappers/Python/ccpi/astra/processors/AstraBackProjector3D.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            raise ValueError("Expected input dimensions is 3, got {0}"\
+                             .format(dataset.number_of_dimensions))
+        
+    def set_ImageGeometry(self, volume_geometry):
+        self.volume_geometry = volume_geometry
+        
+    def set_AcquisitionGeometry(self, sinogram_geometry):
+        self.sinogram_geometry = sinogram_geometry
+    
+    def process(self):
```
Comment (paskino) [inline Wrappers/Python/ccpi/astra/processors/AstraBackProjector3D.py]: could you add the `out` parameter? see [here](https://github.com/vais-ral/CCPi-Framework/blob/composite_operator_datacontainer/Wrappers/Python/ccpi/framework/framework.py#L1123) and [here](https://github.com/vais-ral/CCPi-Framework/blob/composite_operator_datacontainer/Wrappers/Python/ccpi/framework/framework.py#L1077)

I'd say something minimal like this should do

```python

def process(self, out=None):
    ... the processing returns some ret
    if out not is None:
        out.fill(ret)
```

Comment (paskino) [reply Wrappers/Python/ccpi/astra/processors/AstraBackProjector3D.py]: or even `raise` an `NotImplementedError()` could work.

**Comment by jakobsj** [comment]
OK, I added the out argument to the processors as well as one of the projectors for which it was missing. I think this now closes #11, #16, #17, please have a look.

**Pull request merged**